config/jobs/kubernetes/sig-network/ingress-gce-e2e.yaml

```diff
@@ -177,7 +177,7 @@ presubmits:
 
 periodics:
 - name: ci-ingress-gce-e2e
-  cluster: k8s-infra-prow-build
```

Comment:
we need to move all the e2e to the old cluster, the presubmit too, we can leave the test and lint in the new cluster

Reply:
see Line 147 `pull-ingress-gce-e2e`